languages.adoc

```diff
@@ -18,88 +18,82 @@ SPDX-FileType: DOCUMENTATION
 SPDX-License-Identifier: Apache-2.0
 ----
 
-== Overview
+== Overview 
 
-Language specific uProtocol libraries (_the library_) implement the data model defined in link:../basics/README.adoc[uProtocol Basic Types] in a particular programming language like Java, Rust or C/C++. A library also provides helpers for validating data model instances, serializing UUris to a textual representation as well as builders for correctly instantiating more complex data types. Each library includes the language specific declaration of the abstract interfaces defined in uProtocol link:../up-l1/README.adoc[Layer 1] and link:../up-l2/README.adoc[Layer 2].
+uProtocol Language specific libraries (_libraries_) are written for a given programming language (ex. Java, C++, etc...) to implement the protocol and are the primary library that is used by applications and services (uEs) to communicate with other uEs. The _libraries_ declare the API for the uProtocol Transport Layer (uP-L1) and implement the uProtocol Communication Layer API (uP-L2).
 
-NOTE: All libraries *MUST* be made available under a permissive (OSS) license like Apache 2.0 or MIT and *MUST* be hosted under the https://github.com/eclipse-uprotocol[Eclipse uProtocol GitHub Organization].
 
-== Source Code Structure
+== Requirements
 
-The following sections define the content and structure of these libraries.
+=== Content
 
-* The root namespace (package name) *MUST* be `org.eclipse.uprotocol`
-* Modules defined in table <<content>> below *MUST* be implemented in their own folder (namespace)
-* Each module *SHOULD* separate the implementation of the specification as defined in table <<folders>> below.
+[oft-sid="req~up-language-api~1"]
```

Comment:
Based on my current implementation of the AsciiDoc support for OpenFastTrace, this should also include the `.specitem` label:
```
[.specitem,oft-sid="req~up-language-api~1"]
```
This is true for the other requirements as well.

Reply:
I fixed them all now